hack/kube/overlays/dev/preprocessing-secret.yaml

```diff
@@ -9,6 +9,12 @@ stringData:
     verbosity = 2
 
     sharedPath = "/home/preprocessing/shared"
+    checkDuplicates = true
```

Comment:
Don't we want checkDuplicates to be false in the dev environment?

Reply:
Sure, thanks!